### PR TITLE
starting number_of_seconds program HW3

### PR DESCRIPTION
**Pull request by suyetake**
[#104957050]

### CODE REVIEW

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# use variables in the string outputs
+puts "There are #{seconds_in_minute} seconds in a minute"
+puts "There are #{minutes_in_hour} minutes in an hour"
+puts "There are #{hours_in_day} hours in a day"
+puts "There are #{days_in_week} days in a week"
+
+# calculate magnitude of seconds in one hour/day/week values
+puts 'That means there are:'
+puts "  #{seconds_in_hour = \
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: You could pull this out into a `seconds_in_hour` variable.

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  seconds_in_minute * minutes_in_hour} seconds in an hour,"
+puts "  #{seconds_in_day = \
+  seconds_in_hour * hours_in_day} seconds in a day,"
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: Please pull out a variable...

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+puts "  #{seconds_in_week = \
+  seconds_in_day * days_in_week} seconds in a week"
+
+# calculate seconds in a year leveraging above calcs
+seconds_in_year = seconds_in_week * weeks_in_year
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb:38]: :+1:

Comment (suyetake) [inline numbers_letters_and_variables/number_of_seconds.rb:38]: I used a new variable within a puts statement, and rubocop complained when I did it elsewhere as a “useless variable” until I did the assignment outside of the puts line of code.

In this particular statement, rubocop did not complain so I left it in.

Is it a bad practice?

File: numbers_letters_and_variables/hello_world.rb [hunk 4]
```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env ruby
```
Comment (jasonnoble) [inline numbers_letters_and_variables/hello_world.rb:1]: This shouldn't be removed...

Comment (suyetake) [inline numbers_letters_and_variables/hello_world.rb:1]: check check..

omitted the prior commit from the rebase.

File: numbers_letters_and_variables/number_of_seconds.rb (hunk 3 above)
Comment (wb97531) [inline numbers_letters_and_variables/number_of_seconds.rb:38]: What line number are you referring to when you say this particular statement?

**Pull request merged**